scripts/docker_setup.sh

```diff
@@ -98,10 +100,61 @@ function move_ifaces() {
 			sudo ip netns exec pause ethtool -N "${ifaces[$i]}" flow-type tcp4 action 0
 			sudo ip netns exec pause ethtool -u "${ifaces[$i]}"
 		fi
+		if [ "$mode" == 'cndp' ]; then
+			# num queues
+			num_q=1
+			# start queue index
+			start_q_idx=22
+			# RSS using TC filter
+			setup_tc "${ifaces[$i]}" $num_q $start_q_idx
+		fi
 	done
 	setup_addrs
 }
 
+# Setup TC
+# Parameters: $1 = interface, $2 = number of queues, $3 = start queue index
+function setup_tc() {
```

Comment:
Should we add a note clarifying/stating that this function is used only for `CNDP`?